jabsrv-cli/build.gradle.kts

```diff
@@ -27,10 +25,16 @@ application{
     )
 }
 
+val javafxVersion = "24.0.1"
```

Comment:
Can this be moved to common build.gradle.kts or is the variable unreachable then?

Reply:
IDK if dependabot will know it then...